frontend/src/scenes/persons/Cohorts.tsx

```diff
@@ -154,7 +154,7 @@ function _Cohorts(): JSX.Element {
                     loading={cohortsLoading}
                     rowKey="id"
                     pagination={{ pageSize: 100, hideOnSinglePage: true }}
-                    rowClassName={'cursor-pointer ' + rrwebBlockClass}
+                    rowClassName={'cursor-pointer ' + 'ph-no-capture'}
```

Comment:
Same as feature flags, think the only thing worth obscuring is the number of users in cohort, and at most the cohort name.